packages/core/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.1
+
+ - chore(release): version core package
+
```

Comment:
This seems to be out of order. The most recent should be on top.

Reply:
I had to do manual versioning at first so my mistake.